.github/pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+### Description
+
+_Short summary of what is the issue and the solution._
+
+### Dependencies
+
+- _Ex. Other PRs._
+
+### Breaking Change
+
+- [ ] Yes
+- [ ] No
+
+### Visuals
+
+_GIFs or screenshots. Before and After screenshots are encouraged._
```

Comment:
Not sure if visuals are relevant for most PR's ?

Reply:
Yeah I don't use that Visuals field a lot. And of course when we want we still can add screenshots to Description field. So I think it is better to remove that. Pushing a fix now.